### ISSUE TITLE
[Feature] Add multi-account support

### ISSUE THREAD

**Issue by andrewsmm** [richsage/RMSPushNotificationsBundle#116]
What do you think about this idea? 
It will be really cool and can be used for sending push notify to more apps with difference accounts.

**Comment by sedigo** [comment]
Absolutely. that's a very interesting feature this library should have. I think I am motivated enough to participate in developing that.

**Comment by sedigo** [comment]
Have you checked in Service/OS/...notifcation.php it is clear that you can pass parameters for multiple accounts
